package.json

```diff
@@ -21,16 +21,16 @@
     "node-api"
   ],
   "dependencies": {
-    "cmake-js": "^7.2.1",
+    "cmake-js": "^7.3.0",
     "detect-libc": "^2.0.2",
     "each-series-async": "^1.0.1",
     "execspawn": "^1.0.1",
     "ghreleases": "^3.0.2",
     "github-from-package": "0.0.0",
-    "glob": "^7.2.3",
+    "glob": "^10.3.10",
```

Comment:
`glob` versions 9+ use a Promise-based API so this upgrade will require a code change (and highlights a possible unit test blind spot).
https://github.com/prebuild/prebuild/blob/f8d82a1d9f1ae17e2a389414d242e22ee8d2efde/bin.js#L45

Reply:
Thanks for spotting that! I moved it to a separate PR (depends on this one though).

I don't know how to test it without actually uploading stuff :)